playbooks/retire_host.yml

```diff
@@ -38,7 +38,13 @@
     # This catches the case where tracking.log is 0b
     - name: Sync again
       command: /edx/bin/send-logs-to-object-store -d "{{ COMMON_LOG_DIR }}/tracking/" -b "{{ COMMON_OBJECT_STORE_LOG_SYNC_BUCKET }}/logs/tracking"
-    
+    - name: Sync edX services logs again
+      command: "/edx/bin/send-logs-to-object-store -d {{ COMMON_LOG_DIR }}{{ item.src }} -b {{ COMMON_OBJECT_STORE_LOG_SYNC_BUCKET }} -p {{ item.dest }}"
+      with_items:
+        - { src: '/lms/',  dest: '{{ COMMON_OBJECT_STORE_EDX_LOG_SYNC_PREFIX }}lms/' }
```

Comment:
@swalladge This line is missing from this [PR](https://github.com/open-craft/configuration/pull/167/files):
```
        - { src: '/tracking/',  dest: '{{ COMMON_OBJECT_STORE_LOG_SYNC_PREFIX }}' }
```

Reply:
@michaelbukachi interesting - looks like this line was removed in this commit: https://github.com/open-craft/configuration/pull/175/commits/3451ed1e48d2ec9ffad0843ffc7e7a13f9ab6222  I can't link that commit to a PR though - however it does look like a follow up commit to the PR you linked. CC @pkulkark

Reply:
@swalladge noted.